app/assets/stylesheets/objects/_side.scss

```diff
@@ -41,7 +41,7 @@
 
     &:hover,
     &:active {
-      color: govuk-colour("bright-red");
+      color: govuk-colour("red");
```

Comment:
All the other changes in this PR are to do with removing the legacy stuff, but this change looks like it's just fixing a color that shouldn't have been used in the first place. Do you mind splitting this one out into a separate commit?

Reply:
This should've been `govuk-colour("red", $legacy: "bright-red")`, but because the legacy colour wasn't declared instead of only removing the legacy declaration it looks like a colour swap. More details about [how the update path for colours on govuk-frontend](https://github.com/alphagov/govuk-frontend/blob/master/CHANGELOG.md#update-colours).

Reply:
Ah, makes sense 👍